### PR TITLE
#1194 Soft Retire NOTES - Approved Progs

### PR DESCRIPTION
**Pull request by C-Love**

### CODE REVIEW

File: notes/approved-programs.vbs [hunk 1]
```diff
@@ -134,7 +135,7 @@ Do
...
+            run_elig_summ = MsgBox("* * * THIS SCRIPT WILL BE RETIRED ON 3/13/2023 * * *" & vbCr & "Please start using Eligibility Summary for approvals of eligibile benefits right away." & vbCr & vbCr &"Run NEW Script - NOTES - Eligibliity Summary?"& vbCr & vbCr & "It appears you are running 'NOTES - Denied Programs' on a case that may be supported by the new script 'NOTES - Eligibility Summary', it is available to use to document the eligibility results denials on SNAP, CASH, HC, and EMER." & vbCr & vbCr & "The script can redirect to run NOTES - Eligibility Summary now. Remember this new script takes some time to gather the details of the approval, but reqquires little input." & vbCr & vbCr & "NOTE: Information entered in this first dialog will NOT carry through." & vbCr & vbCr & "Would you like the script to run NOTES - Eligibility Summary for you now?", vbQuestion + vbYesNo, "Redirect to NOTES - Eligibility Summary")
```
Comment (IlseFerris) [inline notes/approved-programs.vbs:138]: Update retirement date. This says 03/13/23.

File: notes/approved-programs.vbs [hunk 2]
```diff
@@ -54,6 +54,7 @@ changelog = array()
 
 'INSERT ACTUAL CHANGES HERE, WITH PARAMETERS DATE, DESCRIPTION, AND SCRIPTWRITER. **ENSURE THE MOST RECENT CHANGE GOES ON TOP!!**
 'Example: call changelog_update("01/01/2000", "The script has been updated to fix a typo on the initial dialog.", "Jane Public, Oak County")
+call changelog_update("03/03/2023", "* * * THIS SCRIPT IS BEING RETIRED ON 05/08/2023 * * *##~####~##Be sure to try using NOTES - Eligibility Summary before this retirement date for CASE/NOTEs on denials. This is the time to become accustomeed to the functionality of NOTES - Eligibility Summary.##~##", "Casey Love, Hennepin County")
```
Comment (IlseFerris) [inline notes/approved-programs.vbs:57]: Change the changelog date to current date.

**Review by IlseFerris — CHANGES_REQUESTED** [review]
Just a quick update on the dates.

**Comment by C-Love** [comment]
You mean, you want the RIGHT dates? That's a lot.

**Review by IlseFerris — APPROVED** [review]
Looks good. I approve thee!

**Pull request merged**